src/provider/ehr_launch.jl

```diff
@@ -203,7 +203,7 @@ end
 function provider_ehr_launch_part_three(
         config::ProviderEHRLaunchConfig,
         location_queryparams::AbstractDict;
-        launch_token::String = "",
+        launch_token::String = "", #used for debugging in case of epic, might not be present in other ehrs
```

Comment:
Probably best not to hardcode a specific EHR; I'd say something like "used for debugging in case of some EHRs; won't be present in all EHRs"